axum/src/extract/ws.rs

```diff
@@ -282,10 +282,7 @@ impl<F> WebSocketUpgrade<F> {
 
     /// Finalize upgrading the connection and call the provided callback with
     /// the stream.
-    ///
-    /// When using `WebSocketUpgrade`, the response produced by this method
-    /// should be returned from the handler. See the [module docs](self) for an
-    /// example.
+    #[must_use]
```

Comment:
What do you think about

```suggestion
    #[must_use = "to setup the WebSocket connection this response must be returned"]
```

Reply:
Looks good, with extra comma :)